nile-backend/src/main/java/com/tp/Nile/controllers/requests/AddProductRequest.java

```diff
@@ -0,0 +1,28 @@
+package com.tp.Nile.controllers.requests;
+
+import lombok.Getter;
+import lombok.Setter;
+
+import java.util.List;
+
+@Setter
+@Getter
+public class AddProductRequest {
```

Comment:
Why use this instead of a Product object with the attributes you need?

Reply:
With the Product model, you would have to enter all the associated info such as features and photos when you add a product. The AddProductRequest is a work around to only enter the ids of features and photos when you add a product